code/datums/supplypacks.dm

```diff
@@ -1411,7 +1411,9 @@ GLOBAL_LIST_INIT(all_supply_groups, list(SUPPLY_EMERGENCY,SUPPLY_SECURITY,SUPPLY
 
 /datum/supply_packs/misc/lasertag
 	name = "Laser Tag Crate"
-	contains = list(/obj/item/gun/energy/laser/tag/red,
+	contains = list(/obj/item/beach_ball/dodgeball,
+					/obj/item/beach_ball/dodgeball,
+					/obj/item/gun/energy/laser/tag/red,
 					/obj/item/gun/energy/laser/tag/red,
```

Comment:
This is gonna conflict with the economy rework

Reply:
I could add it to the crate later but...exchanging the beach ball for the baseball one is necessary, no problem on being on hold.